README.md

```diff
@@ -14,46 +14,65 @@ ____
-| Download |[![Download](http://i.imgur.com/odToka3.png)](https://github.com/tidusjar/Ombi/releases)             |      [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/eap/artifacts)       |  [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/dev/artifacts)            |[![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore/artifacts)            |
+| Service  | Master (Version 2.0 - Recommended)                | Open Beta     (Version 3.0)          |
+|----------|:---------------------------:|:----------------------------:|
+| AppVeyor | [![Build status](https://ci.appveyor.com/api/projects/status/hgj8j6lcea7j0yhn/branch/master?svg=true)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/master) | [![Build status](https://ci.appveyor.com/api/projects/status/hgj8j6lcea7j0yhn/branch/DotNetCore?svg=true)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore) | 
+| Download |[![Download](http://i.imgur.com/odToka3.png)](https://github.com/tidusjar/Ombi/releases)            |      [![Download](http://i.imgur.com/odToka3.png)](https://ci.appveyor.com/project/tidusjar/requestplex/branch/DotNetCore/artifacts)       | 
 # Features
-Here some of the features Ombi has:
-* All your users to Request Movies, TV Shows (Whole series, whole seaons or even single episodes!) and Albums
+Here some of the features Ombi V3 has:
+* Now working without crashes on Linux.
+* Lets users request Movies and TV Shows (whether it being the entire series, an entire season, or even single episodes.)
 * Easily manage your requests
-* User management system (supports plex.tv accounts and local accounts)
-* Sending newsletters
-* Fault Queue for requests (Buffer requests if Sonar/Couchpotato/SickRage is offline)
-* Allow your users to report issues and manage them separately
-* A landing page that will give you the availability of your Plex server and also add custom notification text to inform your users of downtime.
-* Allow your users to get notifications!
+* User management system (supports plex.tv, Emby and local accounts)
+* A landing page that will give you the availability of your Plex/Emby server and also add custom notification text to inform your users of downtime.
+* Allow your users to get custom notifications!
 * Secure authentication so you don't have to worry about those script kiddies
-* We check to see if the request is already in Plex, if it's already in Plex then why you requesting it?!
-* We have allowed the ability for a user to add a custom note on a request
-* It automatically updates the status of requests when they are available on Plex
+* We check to see if the request is already on Plex, if it's already on Plex then why you requesting it?!
+* It automatically updates the status of requests when they are available on Plex/Emby
```

Comment:
Automatically updates the status of requests when they are available on Plex/Emby